Dockerfile

```
@@ -7,14 +7,12 @@ WORKDIR /app
 
 COPY package.json .
 
-RUN apk add --no-cache --virtual .build-dependencies git && \
-    sed -i -e 's/"mongojs": "0.18.2"/"mongojs": "2.4.0"/g' package.json && \
-    yarn install && \
-    yarn cache clean && \
+RUN apk add --no-cache --virtual .build-dependencies git python make && \
+    npm install && \
+    npm cache clean --force && \
     apk del .build-dependencies
 
 USER node
 
 COPY . .
-
-CMD ["npm", "start"]
+CMD ["node", "lib/index.js"]
```

I don't understand the purpose of this change (absent from tidepool merge)  
```suggestion
CMD ["npm", "start"]
```